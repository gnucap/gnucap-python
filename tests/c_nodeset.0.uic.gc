* Felix Salfelder 2019
* GPLv3+

attach public c_python.so
loadpy c_nodeset.py

options numdgt=10

verilog

resistor #(.r(1)) r1(0,1);
capacitor #(.c(1)) c2(1,0);
nodesetter #(.ic0(1.7)) ns(1);

print dc v(nodes) iter(0) df(ns.vs0) f(ns.vs0)
print tran v(nodes) iter(0) df(ns.vs0) f(ns.vs0)

dc
* seems to be incomplete
transient 1 uic
