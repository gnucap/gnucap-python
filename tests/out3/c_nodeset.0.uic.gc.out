Gnucap : The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2013, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
main version
core-lib version
default plugins
generator: already installed, replacing
stashing as generator:0
transient: already installed, replacing
stashing as transient:0
transient: already installed, replacing
stashing as transient:0
print dc v(nodes) iter(0) df(ns.1) f(ns.1) 
                             ^ ? no match
print dc v(nodes) iter(0) df(ns.1) f(ns.1) 
                                     ^ ? no match
print tran v(nodes) iter(0) df(ns.1) f(ns.1) 
                               ^ ? no match
print tran v(nodes) iter(0) df(ns.1) f(ns.1) 
                                       ^ ? no match
#                v(1)            iter(0)        
 0.              1.699983        3.             
#Time            v(1)            iter(0)        
 0.              0.              0.             
 0.01            0.              3.             
 1.              0.              3.             

